app/src/index.ts

```diff
@@ -77,7 +77,7 @@ const configuration: ConfigurationContext = {
   disableOnboardingSkip: true,
   useCustomNotifications: useNotifications,
   useAttestation: useAttestation,
-  enableUseMultUseInvitation: true,
+  enableUseMultUseInvitation: false,
```

Comment:
Did I just recently see a PR go through that changed this value?

Reply:
yes, that change needs to be reverted as it's part of the cause of the duplicate connection issue